js/babel-plugin-sprockets-commoner-internal/index.js

```diff
@@ -198,6 +251,25 @@ module.exports = function (context) {
         }
         break;
       }
+    },
+    Identifier: function Identifier(path, state) {
+      if (
+        (path.node.name === 'process' || path.node.name === 'Buffer')
+        && (
+          path.parent.type !== 'MemberExpression'
+          || path.parentKey === 'object'
+        )
+        && !path.scope.hasBinding(path.node.name)
```

Comment:
Could probably use a test for this part of things as well, unless I missed it below.